hubblestack/hec/dq.py

```diff
@@ -0,0 +1,127 @@
+
+import time
+from diskcache import Deque
+from sqlite3 import DatabaseError as SQLiteDBError
+import shutil, os
+import glob
+
+# fakelogging goes to a file iff specified in environment
+# and otherwise does nothing (for debugging)
+import fakelogging as logging
+log = logging.getLogger('dq')
```

Comment:
I think using `__name__` is the standard way to do this -- why hardcode it if you don't have to?